src/syscheckd/src/create_db.c

```diff
@@ -555,8 +709,12 @@ static void _fim_file(const char *path,
     }
 
     if (txn_handle != NULL) {
+        txn_context->latest_entry = &new;
```

Comment:
There is a problem here when you run cppcheck, the message is:
```
wazuh/src/syscheckd/src/create_db.c:606:9: error: Address of local auto-variable assigned to a function parameter. [autoVariables]
txn_context->latest_entry = &new
```

Reply:
I think that we are running cppcheck in C code. This is not a problem with the code itself, as new is a reserved keyword for C++ but not for C. I can change the name of the variable to avoid this false positive.

Reply:
Add `*:*src/syscheckd/src/create_db.c:712` inside cppcheckSuppress.txt if this is not an error because without this change the RTR is failing